templates/app/onboard-guide/index.html.twig

```diff
@@ -42,15 +42,16 @@
         </div>
         <div class="slide fade-in slide-in">
             <div class="header">
-                <h1>{{ get_configuration('app_terms_header')|default('Accept terms'|trans) }}</h1>
+                {# TODO: Get configutation to work so we dont need to comment out the below code #}
+                {# <h1>{{ get_configuration('app_terms_header')|default('Accept terms'|trans) }}</h1>
                 {% if get_configuration('app_terms_lead')|default(null) %}
                     <p class="lead">{{ get_configuration('app_terms_lead') }}</p>
-                {% endif %}
+                {% endif %} #}
```

Comment:
This works on my machine after a clean install (and loading fixtures). I think we can safely remove the comment.